checkov/terraform/tf_parser.py

```diff
@@ -241,10 +241,15 @@ def _load_modules(self, root_dir: str, module_loader_registry: ModuleLoaderRegis
                         resolved_loc_list = self.module_to_resolved[current_nested_data]
                     self.module_to_resolved[current_nested_data] = resolved_loc_list
 
-                    specified_vars = {k: v[0] if isinstance(v, list) and v else v for k, v in module_call_data.items()
-                                      if k != "source" and k != "version"}
-                    skipped_a_module = self.should_skip_a_module(specified_vars, ignore_unresolved_params)
-                    if skipped_a_module:
+                    specified_vars = {
+                        k: v[0] if isinstance(v, list) and v else v
```

Comment:
this can probably be optimised,
we're iterating the values twice

Reply:
yeah, there are for sure some optimization possibilities in this area 🙈